packages/server/api/src/app/pieces/piece-metadata-service/hooks/piece-filtering.ts

```diff
@@ -1,51 +1,61 @@
 import { PieceCategory } from '@activepieces/shared'
 import { PieceMetadataSchema } from '../../piece-metadata-entity'
 import Fuse from 'fuse.js'
+import { ActionBase, TriggerBase } from '@activepieces/pieces-framework'
+
+
+const defaultFilterKeys = [  {
```

Comment:
REmove it and replace it in line ['name', 'description'] also no need for wieght if they are same